csharp/src/Ice/InputStream.cs

```diff
@@ -236,7 +238,6 @@ private void initialize(Ice.Internal.Instance instance, EncodingVersion encoding
         _classGraphDepthMax = _instance.classGraphDepthMax();
         _valueFactoryManager = _instance.initializationData().valueFactoryManager;
         _logger = _instance.initializationData().logger;
```

Comment:
You'll notice no change to OutputStream and no change to the tests (except for the removal of a metadata directive in the Slice files).